requests-for-comments/2025-01-30-podcast-feedback.md

```diff
@@ -0,0 +1,258 @@
+# Podcast feedback
+
+This document is an exhaustive aggregate of all the feedback we received from recording initial episodes of our PostHog podcast. It's intended to turn all that feedback into actionable takeaways and a list of decisions we need to make about the direction for future episodes.
+
+For context, we set out to produce a podcast as part of our Q1 marketing goals. But there has been some internal debate about whether we should:
+
+- soft launch and iterate over time to produce higher quality (see the parable about how [quantity leads to quantity](https://austinkleon.com/2020/12/10/quantity-leads-to-quality-the-origin-of-a-parable)), or...
+- continue producing internal-only episodes as we improve the content until it's good enough to launch publicly
+
+## Decisions
+
+### Should we release any of the content we have so far?
+
+**Recommendation:** Aggressively edit the content from the episodes with Cory and Raquel. We have about 50 minutes for each episode, but we can cut out content that didn't land and try to aim for tighter episodes closer to 20 minutes in length that are made up of the best content we recorded.
+
+### Should we do audio and video? Or consider starting with audio only?
+
+**Recommendation:** Starting with both audio and video was likely a mistake since we ended up with too many variables to worry about. Having said that, my recommendation is that we stick with both. Part of the original goals for the podcast were to open the door to video content and work towards improving our ability to produce video content internally.
+
+### Should we have separate edits for audio and video?
+
+**Recommendation:** Long term, definitely. But in the short term, we should focus on aggressively editing content and keeping it tight even if the video versions end up feeling a bit janky.
+
+### How long should each episode be?
+
+**Recommendation:** The shorter the better. We should aim for 10-20 minutes, with a maximum of 30 minutes only if we have a ton of great content.
+
+### What should we name the show? Something obvious like Product Engineering?
+
+**Recommendation:** I prefer "The Product Engineering Podcast" since it's obvious in terms of the value proposition. But a lot of people liked "Quick Calls with Jimmy Hawkins" and I don't feel strongly either way.
```

Comment:
We haven't tried that hard to find a name yet, so we should explore more options. My take is "The Product Engineering Podcast" is a bit long-winded, and Quick Calls is funny at first, but a bit too vague.